zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/archiver/OpenSearchRepositoryIT.java

```diff
@@ -81,6 +87,81 @@ void shouldDeleteDocuments() throws IOException {
         .isEqualTo("3");
   }
 
+  @Test
+  void shouldSetIndexLifeCycle() throws IOException {
+    // given
+    final var indexName = UUID.randomUUID().toString();
+    final var repository = createRepository();
+    testClient.indices().create(r -> r.index(indexName));
+
+    final OpensearchEngineClient engineClient = new OpensearchEngineClient(testClient);
+    final OpenSearchGenericClient genericClient =
+        new OpenSearchGenericClient(testClient._transport(), testClient._transportOptions());
```

Comment:
:wrench: Both can be `final var`, because we're constructed the object directly, so the type is obvious from the right-hand-side of the declaration. It's better not to use var only if the type is not really obvious.